src/InkRouter/Models/Side.php

```diff
@@ -234,6 +251,44 @@ public function getPrintAssets()
     }
 
     /**
+     * @return bool
+     */
+    public function isVariableUv()
+    {
+        return $this->variableUv;
+    }
+
+    /**
+     * @param bool $variableUv
+     * @return InkRouter_Models_Side
+     */
+    public function setVariableUv($variableUv)
+    {
+        $this->variableUv = $variableUv;
```

Comment:
i don't think it's a good idea to make it inconsistent with current file specs in xml.
why would we need bool? if url is there, then order has variable uv. if they really want it, why don't we have it for spot?
also, we must have file hash to handle order updates.
please contact BAs/profit team again.